runbuild.sh

```diff
@@ -4,4 +4,6 @@
 which cargo >/dev/null || . "$HOME/.cargo/env"
 
 cargo build --release
-cd rtc_data_enclave && make
+cd rtc_data_enclave && make && cd ..
+cd rtc_auth_enclave && make && cd ..
+cd rtc_exec_enclave && make && cd ..
```

Comment:
A bit easier and more idiomatic:

```suggestion
(cd rtc_data_enclave && make)
(cd rtc_auth_enclave && make)
(cd rtc_exec_enclave && make)
```